custom-connectors/PUGGamifiedEngagement/Readme.md

```diff
@@ -0,0 +1,68 @@
+## Picnic Connector
```

Comment:
@smart-jp Please move your connector's folder from `custom-connectors` folder to `certified-connectors` folder as you have submitted your connector for certification.

Reply:
Commited. Thanks!